### PR TITLE
add ipv6 for skidoo

### PR DESCRIPTION
**Pull request by novacain1**
This PR:

1. Adds IPv6 support for the Skidoo Single Node OpenShift cluster.  We should always test with IPv6
2. Streamlines semantics of siteconfig a bit, removes quotes where not necessary
3. Adds VSE ssh key

### CODE REVIEW

**Review by natifridman — CHANGES_REQUESTED** [review]
Please fix lint errors

**Review by natifridman — APPROVED** [review]
LGTM, please wait for #54 before merge. I would like to test the cherry-pick in this PR

**Review by nocturnalastro — APPROVED** [review]

**Pull request merged**
